### PR TITLE
Add Loggers

### PR DESCRIPTION
**Pull request by kevinresol**
Implementation for #25.

For flash I am not sure how I can log to the terminal. Looking at [buddy's code](https://github.com/ciscoheat/buddy/blob/54b8ab328e7ba2db4cd985b72f8b897f059d6a5f/src/buddy/internal/sys/Flash.hx) it seems to just create a TextField and write to it. But I am not sure. @ciscoheat can you give me some hints?

### CODE REVIEW

**Comment by ciscoheat** [comment]
That buddy implementation is for the non-CI version. But since travix is setting up everything, the CI implementation shouldn't be more complicated than the one in the instructions: https://github.com/back2dos/travix#building-for-flash

I don't think `print` can be done however, since trace is always outputting with a newline. In buddy, I simply ignore the print output when the target and platform doesn't support it.

**Comment by kevinresol** [comment]
Alright I have implemented the flash logger

**Comment by kevinresol** [comment]
I will take the lead to get this merged, because it shouldn't affect backward compatibility.
We can always improve it later.

**Comment by back2dos** [comment]
:+1:

File: src/travix/loggers/FlashLogger.hx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  
+  static function println(s:String) {
+    var pending = printBuf.toString();
+    flash.Lib.trace(pending + s);
+    if(pending.length > 0)
+      printBuf = new StringBuf();
+  }
+  
+  static function exit(code:Int) {
+    flash.system.System.exit(code);
```
Comment (ciscoheat) [inline src/travix/loggers/FlashLogger.hx]: Flush the printbuf on exit?

Comment (kevinresol) [inline src/travix/loggers/FlashLogger.hx]: Good idea

**Pull request merged**